docs/_locale/de/LC_MESSAGES/settings.po

```diff
@@ -40,9 +40,39 @@ msgstr "Die Nutzerliste zeigt alle Mitglieder in Ihrer Organisation. Sie können
 msgid "Administrators can promote other users to administrator: on an expanded user panel, click the *Promote to administrator* button."
 msgstr "Administratoren können andere Nutzer zu Administratoren ernennen. Klappen Sie dazu einen Eintrag in der Nutzerliste auf und klicken Sie auf *Zum Administrator ernennen*."
 
+msgid "Replacements"
+msgstr "Vertreter"
+
 msgid "Administrators can also delete users, using the delete button next to the user name."
 msgstr "Administratoren können Nutzer auch löschen indem Sie auf den Papierkorb neben dem Nutzernamen klicken."
 
+msgid "If you want to delete a user account, for example, because the user is leaving your organization, unfinished tasks to this account may still exist. In this case, specify a replacement who will take over open task assignements. The substitute will then have exactly all access rights to complete these tasks, but not automatically inherits all rolls or rights of the deleted user."
+msgstr "Wenn Sie ein Benutzerkonto löschen möchten, da der Nutzer beispielsweise Ihre Organisation verlässt, könnten noch unerledigte Aufgaben vorliegen. In diesem Fall haben Sie die Möglichkeit, einen Vertreter zu bestimmen, der diese offenen Aufgaben übernehmen soll. Der Vertreter erhält dann genau die Zugriffsrechte, die er für die Erledigung der Aufgabe benötigt, erbt jedoch nicht automatisch Rollen und Rechte des gelöschten Nutzers."
+
+msgid "The substitute takes over"
+msgstr "Der Vertreter übernimmt"
+
+msgid "in workflows: owner, assignments, candidates, default values for form fields, transition conditions, JavaScript test values, access"
```

Comment:
Do you usually call it `access` or `access rights`. The same in german `Zugriff` vs. `Zugriffsrechte`.